docs/overview/architecture.md

```diff
@@ -111,7 +111,37 @@ When the user searches a vector from Vald:
 15. Vald Egress Filter will return the filtered result to the Vald Filter Gateway.
 16. Vald Filter Gateway will return the final result to the Vald Ingress.
 
-<!-- ### Update -->
+### Update
+
+<img src="../../assets/docs/update_flow.png" />
+
+When the user updates a vector from Vald:
+
+1. Vald Ingress receives the request from the user. The request includes the new vector ID(s) and the vector(s) to be updated.
```

Comment:
```suggestion
1. Vald Ingress receives the request from the user. The request includes the vector ID(s) and the new vector(s) to be updated.
```
Is this right?

Reply:
how about this?
```suggestion
1. Vald Ingress receives the request from the user. The request includes the existing vector ID(s) and the new vector(s) to be updated.
```

Reply:
I think your suggestion seems like strict correct. 👍